bindings/nodejs/examples/1-create-account.js

```diff
@@ -3,42 +3,17 @@
  */
 
 require('dotenv').config();
+const { CoinType } = require('../out/types');
 const manager = require('./account-manager');
 
 async function run() {
-    const { initLogger } = require('@iota/wallet');
-
-    initLogger({
-        color_enabled: true,
-        outputs: [{
-            name: './wallet.log',
-            level_filter: 'debug'
-        }]
-    })
-    
-    const manager = new AccountManager({
```

Comment:
This should stay?

Reply:
Because it was duplicated in every example. I thought we could just enable it by default

Reply:
Will approve now, but soon it will not be necessary anymore to add all the fields every time and then it shouldn't be moved to another file I think